docker/run-build-in-docker.sh

```diff
@@ -9,16 +9,16 @@ TARGET_DIR=`mktemp -d`
 git clone --depth=1 "$RUST_NANOS_SDK_GIT" "$RUST_NANOS_SDK"
 cd "$RUST_NANOS_SDK"; git checkout "$RUST_NANOS_SDK_REV"; cd -
 
-PATH=$RUST_NANOS_SDK/ledger_device_sdk:$PATH
+PATH=$RUST_NANOS_SDK/ledger-device-rust-sdk:$PATH
```

Comment:
Did we previously forget to change this?

Reply:
No, they recently moved the files https://github.com/LedgerHQ/ledger-device-rust-sdk/commit/7c47bb9cdbec19e0b3d0a4742bb833a36cab6431